src/lib/swapper/api.ts

```diff
@@ -294,6 +294,8 @@ export type GetUnsignedTxArgs = {
   accountMetadata?: AccountMetadata
   stepIndex: number
   supportsEIP1559: boolean
+  buyAssetUsdRate: string
```

Comment:
Needed to remove circular dependancies.

Reply:
is this still the case with thorchain commented out?

Reply:
It is, unfortunately!